python/tvm/contrib/pytorch.py

```diff
@@ -0,0 +1,57 @@
+"""PyTorch bridge wrapping torch tensor."""
+
+import ctypes
+from decorator import decorate
+from .. import register_extension, TypeCode
+
+def to_pytorch(module):
+    """Wrap a TVM function as PyTorch function
+
+    Parameters
+    ----------
+    func : Function
+        A TVM function that can take positional arguments
+
+    args : Tuple of arguments
+        Tuple of arguments to pass to the wrapped function
+
+    Returns
+    -------
+    func : Function
+        A function that can take PyTorch tensor as argument
+        in places that used to expect TVM NDArray.
+    """
+    #import pytorch, check for pytorch tensor
+    import torch
+    def converter(func, *args):
+        args = tuple([FireTensor(arg) if isinstance(arg, torch.Tensor) else arg for arg in args])
+        return func(*args)
+
+    def wrapper(*args):
+        module(*args)
+
+    return decorate(wrapper, converter)
```

Comment:
decorator is needed to preserve function signature. Since we are doing wrapping anyway, we do not need the decorator here, just do

```python
def to_pytorch(func):
    def _wrapper(*args):
         args = tuple(DLManagedTensor(torch.C.to_dlpack(arg)) if isinstance(arg, torch.Tensor) else arg for arg in args)
         return func(*args)
    return _wrapper

```